marklogic-data-hub/build.gradle

```diff
@@ -40,6 +40,10 @@ ext {
     thirdPartySourcePath = "src/main/resources/ml-modules/root/data-hub/third-party"
 }
 
+configurations {
+    datahubLogging
```

Comment:
Sorry one more thing - can you add a comment on this explaining why it exists? We want to document that this is used so we can include a logging implementation jar in the application artifacts produced by this project - the installer and the client jars - without that ending up in the library artifact.